docs/configuration.asciidoc

```diff
@@ -221,6 +221,23 @@ A lower value will increase the load on your APM server,
 while a higher value can increase the memory pressure of your app.
 A higher value also impacts the time until transactions are indexed and searchable in Elasticsearch.
 
+
+[float]
+[[config-max-event-queue-length]]
+==== `max_event_queue_length`
+
+|============
+| Environment                          | Django/Flask             | Default
+| `ELASTIC_APM_MAX_EVENT_QUEUE_LENGTH` | `MAX_EVENT_QUEUE_LENGTH` | `None`
+|============
+
+Maximum queue length of transactions before sending transactions to the APM server.
+A lower value will increase the load on your APM server,
+while a higher value can increase the memory pressure of your app.
+A higher value also impacts the time until transactions are indexed and searchable in Elasticsearch.
+
+This setting is useful to limit memory consumption if you experience a sudden spike of traffic.
```

Comment:
Since there is no default value, could you add a note or link to the description of what the default value for sending events is?